src/gui/windows/debug.zig

```diff
@@ -39,7 +39,23 @@ pub fn render() void {
 	draw.print("window size: {}×{}", .{main.Window.width, main.Window.height}, 0, y, 8, .left);
 	y += 8;
 	if (main.game.world != null) {
-		draw.print("Pos: {d:.1}", .{main.game.Player.getPosBlocking()}, 0, y, 8, .left);
+		const player = main.game.Player;
+		draw.print("Pos: {d:.1}", .{player.getPosBlocking()}, 0, y, 8, .left);
+		y += 8;
+		draw.print("IsFlying: {} IsGhost: {} HyperSpeed: {}", .{
+			player.isFlying.raw,
```

Comment:
These are atomics. You are not supposed to access the raw values directly.
While in this case it's not an actual issue, it still shows up in the thread sanitizer, making it harder to debug actual data races.
Instead you should use `.load()`, and in this case, since you don't care about memory ordering, you can just use `.unordered`

Reply:
Ah, that does make sense, fixed it in 505d1f2. 
I noticed that I didn't lock the mutex when getting onGround and jumpCooldown, should that also be fixed?
I could potentially just lock the mutex once and get all the values at the same time before drawing them, then unlock it.